Fix/separation (#1)
qbeyond/terraform-google-folder: README.md

| [google_storage_bucket_iam_member.gcs-sinks-binding](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/storage_bucket_iam_member) | resource |
### main.tf
| Name | Type |
|------|------|
| [google-beta_google_essential_contacts_contact.contact](https://registry.terraform.io/providers/hashicorp/google-beta/latest/docs/resources/google_essential_contacts_contact) | resource |
| [google_folder.folder](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/folder) | resource |
| [google_folder.folder](https://registry.terraform.io/providers/hashicorp/google/latest/docs/data-sources/folder) | data source |
### organization-policies.tf
| Name | Type |
|------|------|
| [google_org_policy_policy.default](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/org_policy_policy) | resource |
### tags.tf
| Name | Type |
|------|------|
| [google_tags_tag_binding.binding](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/tags_tag_binding) | resource |

> <!-- END_TF_DOCS -->

Reviewer: ```suggestion
<!-- END_TF_DOCS -->

## Contribute

This module is derived from [google cloud foundation fabric module `folder` v19](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/tree/v19.0.0/modules/folder). It is designed to be able to integrate new changes from the base repository. Refer to [guide in `terraform-google-landing-zone` repository](https://github.com/qbeyond/terraform-google-landing-zone/tree/main#updating-a-repository) for information on integrating changes.
```

| [sink_writer_identities](outputs.tf#L45) | Writer identities created for each sink. |  |

> <!-- END TFDOC -->

Reviewer 2: Please remove the now duplicated docs left by google.

<!-- BEGIN_TF_DOCS -->
## Usage

> This Module creates a GCP Folder

Reviewer 2: ```suggestion

## Basic

This Module creates a GCP Folder.
```

```hcl
provider "google" {
  project = var.project_id
  user_project_override = true
  billing_project = var.project_id
}

resource "random_string" "folder_name" {
  length           = 8
  special          = false
  upper            = false
  numeric          = false
}

data "google_organization" "default" {
  domain = var.organization_domain
}

module "folder" {
  source = "../.."
  parent = data.google_organization.default.id
  name   = random_string.folder_name.result
}
variable "project_id" {
  type = string
  description = "The Project ID where to create the Folder"
}

variable "organization_domain" {
  type = string
}
```

> This Module creates a GCP Folder with a firewall policy
> It is possible to include yaml config files.

Reviewer 2: Copied from googles docs. 

```suggestion
### Firewall policy factory

In the same way as for the [organization-configuration](https://github.com/qbeyond/terraform-google-organization-configuration) module, the in-built factory allows you to define a single policy, using one file for rules, and an optional file for CIDR range substitution variables. Remember that non-absolute paths are relative to the root module (the folder where you run `terraform`).

```

Author: done

  special          = false
  upper            = false
  numeric          = false
}

data "google_organization" "default" {
  domain = var.organization_domain
}

module "folder" {
  source = "../.."
  parent = data.google_organization.default.id
  name   = random_string.folder_name.result

  firewall_policy_factory = {
    cidr_file   = "configs/cidrs.yaml"
    policy_name = "test"
    rules_file  = "configs/rules.yaml"
  }
  firewall_policy_association = {
    factory-policy = module.folder.firewall_policy_id["test"]
  }
}
variable "project_id" {
  type = string
  description = "The Project ID where to create the Folder"
}

variable "organization_domain" {
  type = string
}
```

```yaml
rfc1918:
  - 10.0.0.0/8
  - 172.16.0.0/12
  - 192.168.0.0/16
allow-admins:
  description: Access from the admin subnet to all subnets
  direction: INGRESS
  action: allow
  priority: 1000
  ranges:
    - $rfc1918
  ports:
    all: []
  target_resources: null
  enable_logging: false

allow-ssh-from-iap:
  description: Enable SSH from IAP
  direction: INGRESS
  action: allow
  priority: 1002
  ranges:
    - 35.235.240.0/20
  ports:
    tcp: ["22"]
  target_resources: null
  enable_logging: false
```

> This Module creates a GCP Folder with a hierarchical firewall policy

Reviewer 2: ```suggestion
### Hierarchical firewall policies

This example creates a GCP Folder with a hierarchical firewall policy 
```
